tests/ui/places/test_places_gnd_tasks.py

```
@@ -0,0 +1,117 @@
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Test GND places tasks."""
+
+from unittest import mock
+
+from rero_mef.places.gnd.tasks import gnd_get_record
+
+
+@mock.patch("rero_mef.places.gnd.tasks.requests_retry_session")
+def test_gnd_get_record_success(mock_session, app):
```

Ruff (0.15.2)

[warning] Unused function argument: `app`

(ARG001)